src/createPersistoid.js

```diff
@@ -16,7 +16,10 @@ export default function createPersistoid(config: PersistConfig): Persistoid {
     config.keyPrefix !== undefined ? config.keyPrefix : KEY_PREFIX
   }${config.key}`
   const storage = config.storage
-  const serialize = config.serialize === false ? x => x : defaultSerialize
+  let serialize = config.serialize === false ? x => x : defaultSerialize
+  if (typeof config.serialize === 'function') {
+    serialize = config.serialize
+  }
```

Comment:
Should this be written a little bit differently so it's not deciding a value then immediately overwriting it? Something like:

```js
let serialize;
if (config.serialize === true) {
  serialize = defaultSerialize;
} else if (config.serialize === false) {
  serialize = x => x;
} else if (typeof config.serialize === 'function') {
  serialize = config.serialize;
} else {
  throw Error('config.serialize must be true, false, or a function');
}
```

Reply:
I guess there is no `config.serialize === true`, people can just pass `false` or `function`, so maybe...

```js
let serialize;
if (config.serialize === false) {
  serialize = x => x;
} else if (typeof config.serialize === 'function') {
  serialize = config.serialize;
} else {
  serialize = defaultSerialize;
}
```

Reply:
Good Idea! This make sense more. At first I just want to modify a little code to implement the feature I want.
Is this performace better, right?

Reply:
I suggested that for readability, not performance. Overwriting `serialize` after it's set by some other logic seems like it could lead to misunderstandings about what's going on with `serialize` (i.e. someone reading the code might not realize we're going to change the value after setting it).

Thanks for the contribution!